src/js/components/FrameworkConfiguration.js

```diff
@@ -279,32 +297,23 @@ class FrameworkConfiguration extends Component {
       focusField,
       activeTab
     } = this.state;
-    const {
-      packageDetails,
-      deployErrors,
-      formErrors,
-      formData,
-      onFormDataChange,
-      onFormErrorChange
-    } = this.props;
 
     let pageContents;
     if (reviewActive) {
       pageContents = this.getReviewScreen();
     } else {
       pageContents = (
         <FrameworkConfigurationForm
-          packageDetails={packageDetails}
           jsonEditorActive={jsonEditorActive}
-          formData={formData}
-          formErrors={formErrors}
           focusField={focusField}
           activeTab={activeTab}
-          deployErrors={deployErrors}
-          onFormDataChange={onFormDataChange}
-          onFormErrorChange={onFormErrorChange}
           handleActiveTabChange={this.handleActiveTabChange}
           handleFocusFieldChange={this.handleFocusFieldChange}
+          {...Util.omit(this.props, [
```

Comment:
Let's please avoid this where possible - it's much more clear what is happening when it is explicit like before.